app/questionnaire/rules.py

```diff
@@ -36,6 +38,53 @@ def evaluate_rule(when, answer_value):
     return result
 
 
+def evaluate_date_rule(when, answer_store, group_instance, metadata, answer_value):
```

Comment:
bit picky but should be called `evalute_date_comparison_rule`

Reply:
Possibly not as it's no different from evaluate_rule being a comparison of answers/values.